Source/WebCore/page/EventHandler.h

```diff
@@ -229,7 +229,7 @@ class EventHandler {
 #endif
 
 #if ENABLE(IOS_TOUCH_EVENTS)
-    bool dispatchTouchEvent(const PlatformTouchEvent&, const AtomString&, const EventTargetTouchMap&, float, float);
+    void updateTouchLastGlobalPositionAndDelta(unsigned, const IntPoint&, bool, bool);
```

Comment:
> It would be nice to replace the bool flags with named enums (even if they're just private to EventHandler), for readability.

Done! https://github.com/WebKit/WebKit/commit/eed593fcf230c5752fc982cef0b7cbdda782897a

> I think it would also be a bit clearer to use PointerID instead of unsigned for the first argument.

Good idea. I'd opted for `unsigned` because that's how `EventHandler` was tracking touch indices in general, but from what I can tell `PointerID == unsigned` so I'll use `PointerID` instead.